python-sdk/tests/extractors/test_extractor.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+from airflow.models.taskinstance import TaskInstance
+from openlineage.client.run import Dataset as OpenlineageDataset
+
+from astro.constants import FileType
+from astro.extractors.extractor import PythonSDKExtractor
+from astro.extractors.facets import InputFileDatasetFacet, OutputDatabaseDatasetFacet
+from astro.files import File
+from astro.sql.operators.load_file import LoadFileOperator
+from astro.table import Metadata, Table
+
+TEST_FILE_LOCATION = "gs://astro-sdk/workspace/sample_pattern"
+TEST_TABLE = "test-table"
+TEST_INPUT_DATASET_NAMESPACE = "gs://astro-sdk"
+TEST_INPUT_DATASET_NAME = "/workspace/sample_pattern"
+TEST_OUTPUT_DATASET_NAMESPACE = "bigquery://None"
+TEST_OUTPUT_DATASET_NAME = "astro.test-extractor"
+INPUT_STATS = [
+    OpenlineageDataset(
+        namespace=TEST_INPUT_DATASET_NAMESPACE,
+        name=TEST_INPUT_DATASET_NAME,
+        facets=InputFileDatasetFacet(
+            file_size=-1,
+            number_of_files=1,
+            file_type=FileType.CSV,
+            description=None,
+            is_pattern=True,
+            files=["gs://astro-sdk/workspace/sample_pattern.csv"],
+        ),
+    )
+]
+
+OUTPUT_STATS = [
+    OpenlineageDataset(
+        namespace=TEST_OUTPUT_DATASET_NAMESPACE,
+        name=TEST_OUTPUT_DATASET_NAME,
+        facets=OutputDatabaseDatasetFacet(
```

Comment:
This needs to be dict too

Reply:
Added it here https://github.com/astronomer/astro-sdk/pull/1025/commits/53013ecbd79bd73aba37f595599cf756f1a5ea01